test/unit/uk/gov/hmrc/tariffclassificationfrontend/service/CasesService_ReferCaseSpec.scala

```diff
@@ -70,7 +70,7 @@ class CasesService_ReferCaseSpec extends UnitSpec with MockitoSugar with BeforeA
       caseUpdating.status shouldBe CaseStatus.REFERRED
 
       val eventCreated = theEventCreatedFor(connector, caseUpdated)
-      eventCreated.userId shouldBe "operator-id"
+      eventCreated.operator.id shouldBe "operator-id"
```

Comment:
better to assert the whole operator object

Reply:
Done and I also found several other instances of this in the service package